5-0-functional-programming/5-1-1-crazy-lambdas/src/main/java/com/bobocode/fp/CrazyLambdas.java

```diff
@@ -186,7 +186,7 @@ public static LongBinaryOperator longSumOperation() {
     }
 
     /**
-     * Returns {@link Supplier} of {@link Supplier} of {@link Supplier} of {@link String} "WELL DONE".
+     * Returns {@link Supplier} of {@link Supplier} of {@link Supplier} of {@link String} "WELL DONE!".
```

Comment:
@disemen thanks a lot, but this is **why the method is called** `tricky...`
Not sure if you have the same sense of humor, but we do 🙂